### PR TITLE
[Merged by Bors] - chore(Order/Notation): move notation classes from other files

### PR DESCRIPTION
**Pull request by urkud**
With this change (and future similar changes), we can avoid importing heavier files if we only need notation, not lemmas.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Order/Heyting/Basic.lean [hunk 1]
```diff
@@ -57,37 +57,6 @@ variable {ι α β : Type*}
 
 /-! ### Notation -/
```
Comment (sgouezel) [inline Mathlib/Order/Heyting/Basic.lean:58]: Remove?

File: Mathlib/Order/Notation.lean [hunk 2]
```diff
@@ -0,0 +1,122 @@
+/-
+Copyright (c) 2017 Johannes Hölzl. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johannes Hölzl, Yury Kudryashov, Yaël Dillies
+-/
+import Mathlib.Tactic.Basic
+import Mathlib.Tactic.Simps.NotationClass
+import Mathlib.Mathport.Rename
+
+/-!
```
Comment (sgouezel) [inline Mathlib/Order/Notation.lean:10]: Mention also top and bot in the file-level docstring?

**Review by sgouezel — COMMENTED** [review]
Can you explain your motivation for this change in the PR description?

**Comment by semorrison** [comment]
bors merge

**Pull request closed without merging**